### ISSUE TITLE
Management Console: coreconfig.go code understanding and improvement

### ISSUE THREAD

**Issue by galenatjpl** [unity-sds/unity-cs#465]

**Comment by jdrodjpl** [comment]
To answer the questions:

1) Yes, this is the code that kicks off when you hit the "Save" button.

2) At the present, I don't think so. It looks like the Project and Venue values use to be editable but were disabled in the frontend, probably because now we need to define that stuff much earlier in order to set up the proxy, etc. The subnets seem to me like something that we probably don't want people to mess with, and I can't find any examples of how the optional parameters are actually being used by the MC code.

3) I suppose that if establishing MC-wide parameters is something that we want to do at some point, a process like this would be useful. But right now, especially given that none of the fields here are really things that we want people to edit (right?), it seems like kind of a superfluous extra step.

4) Pretty easily -- this used to be invoked during the bootstrap but that part was commented out at some point.

**Comment by galenatjpl** [comment]
thanks @jdrodjpl !  Would the next logical step be to just uncomment this and/or put a conditional flag around this (and have it default to run automatically)?  If so, should we just turn this ticket into making that happen?

**Comment by jdrodjpl** [comment]
Yeah, I think that makes sense. I'll do that and update the ticket.

**Comment by galenatjpl** [comment]
@jdrodjpl just checking in here.  Is the auto-invoke part implemented?  If not, it's okay.  I just noticed the ticket status was not "in progress".  There are probably bigger fish to fry anyway.

**Comment by jl-0** [comment]
@jdrodjpl, checking in on this.  Based on the screenshot this seems very old.  Is any of this still relevant.  Seems like the bootstrap covers all this and not really needed in most cases right.
